rfc6962-bis.xml

```diff
@@ -572,6 +581,15 @@ TBSCertificate from it, redacting the domain name labels indicated by the
 redacted labels extension, and deleting the SCT list extension and redacted
 labels extension.
         </t>
+        <t>
+          <spanx style="verb">id</spanx> identifies a single extension from the IANA registry in <xref target="extension_ids"/>.
```

Comment:
I'm a little bit concerned about possible confusion with "LogID id;", which is defined in the same section.  But I'll think about addressing this in a separate PR.